<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>757</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_title">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="text">
       <string>                                                                                                               La Banque</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>modif_chart</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>modif_date</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit_2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>choisir_agneces</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>agence 4</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>agence 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_5">
            <property name="text">
             <string>agence 5</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>agence 3</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_1">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>agence 1</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>ok</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3names">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>                        ca                                                                                       df                                                                                cb</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QtCharts::QChartView" name="graphicsView"/>
      </item>
      <item row="0" column="1">
       <widget class="QtCharts::QChartView" name="graphicsView_2"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>bars</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QtCharts::QChartView" name="graphicsView_3"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>percent</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>bars</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>percent</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>percent</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>bars</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_pret">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>                                                                                                                 PRÊT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QtCharts::QChartView" name="graphicsView_pret">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="interactive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>info_on_prêt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_retour">
      <property name="text">
       <string>retour_sur_main</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QtCharts/QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
